ui/client/cypress/integration/components.ts

```diff
@@ -1,7 +1,7 @@
 describe("Components list", () => {
   const seed = "components"
 
-  const totalComponents = 93
+  const totalComponents = 90
```

Comment:
do u have concept why it is reduced again? :)

Reply:
Changes in DevProcessConfigCreator, I added base&kafka components to streaming-dev classpath in config